docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -441,36 +441,37 @@ Normal comments, review bodies, reviews' inline comments of GitHub's pull reques
 
 | **field**       | **type**        | **length** | **description**                                                                               | **key**           |
 | :-------------- | :-------------- | :--------- | :-------------------------------------------------------------------------------------------- | :---------------- |
 | `id`            | varchar         | 255        | This key is generated based on details from the original plugin                               | PK                |
 | `name`          | varchar         | 255        | For gitlab, as there is no name for pipeline, so we use projectId, others have their own name |                   |
-| `result`        | varchar         | 100        | The result of this task, e.g. SUCCESS, FAILURE                                                |                   |
-| `status`        | varchar         | 100        | The status of this task, e.g. IN_PROGRESS, DONE                                               |                   |
-| `type`          | varchar         | 100        | To indicate if this is a DEPLOYMENT                                                           |                   |
-| `duration_sec`  | bigint unsigned |            | how long does this task take                                                                  |                   |
-| `created_date`  | datetime        | 3          | when did this task start                                                                      |                   |
-| `finished_date` | datetime        | 3          | when did this task finish                                                                     |                   |
-| `environment`   | varchar         | 255        | To indicate the environment in which the task is running                                      |                   |
+| `result`        | varchar         | 100        | The result of the pipeline. It will be standardized to 'SUCCESS', 'FAILURE' in DevLake based on each plugin's possible results.                                                |                   |
+| `status`        | varchar         | 100        | The status of the pipeline. It will be standardized to 'DONE', 'IN_PROGRESS' in DevLake based on each plugin's possible statues.      |
+| `type`          | varchar         | 100        | The value will be set to 'DEPLOYMENT' if it matched the ReGex configured in the Scope Config, otherwise it is an empty string.                                                            |                   |
+| `environment`   | varchar         | 255        | The value will be set to 'PRODUCTION' if it matched the ReGex configured in the Scope Config, otherwise it is an empty string.                                      |                   |

```

Comment:
I think `ReGex` should be `regex`, https://en.wikipedia.org/wiki/Regular_expression

Reply:
@d4x1 I was trying to use RegEx...